src/i18n/Localizable.xcstrings

```diff
@@ -23131,7 +23251,8 @@
       }
     },
     "Stop" : {
-      "extractionState" : "migrated",
+      "comment" : "a bit ambiguous word. example needed :(",
```

Comment:
"Stop" means to close a running connection

Reply:
changed noun to gerund